README.md

```diff
@@ -984,6 +984,10 @@ in inheritance.
   is a hybrid of `Array`'s intuitive inter-operation facilities and
   `Hash`'s fast lookup.
 
+*  Prefer `map` over `collect`, `detect` over `find`, and `reduce` over
```

Comment:
Yeah, in the original discussion people seemed to feel it was self-evident that `map` was better than `collect`. We didn't talk about `size` v. `length` and/or `count` in that PR but I'm happy to add them as well.